lib/katello.rb

```diff
@@ -6,6 +6,7 @@
 require "oauth"
 require "gettext_i18n_rails"
 require "foreman-tasks"
+require "foreman_remote_execution"
```

Comment:
Given https://github.com/Katello/katello/blob/6d8d3ca36e1469d1f7c2c8e180e42467176ac1a4/katello.gemspec#L37
I am rather confident we never have a Katello install w/o REX.

But at the same time it indicates that the way we load stuff differs so heavily in prod/dev/test, that things go weird.

Reply:
https://community.theforeman.org/t/requiring-rex-in-katello-4-0/19582 indicates that dep is here to stay :)